src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/repository/QueryParams.kt

```diff
@@ -28,15 +28,15 @@ open class QueryParams(private val enumerated: Boolean) {
         params["type"] = value
     }
 
-    fun file() = type("nt:file")
+    fun file() = type(ResourceType.FILE.value)
```

Comment:
like it!

Reply:
or you could add method `type(type: ResourceType)` then `.value` could be skipped in each shorthand method